test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Preseed default.snap.txt

```diff
@@ -55,7 +55,7 @@ user-setup-udeb passwd/make-user boolean false
 tasksel tasksel/first multiselect minimal, ssh-server, openssh-server
 
 # Install some base packages
-d-i pkgsel/include string lsb-release wget
+d-i pkgsel/include string lsb-release wget python
```

Comment:
I think this might be the issue - not sure where the python came from, but it is definitely not related to this change

Reply:
ding ding ding and the winner is:

https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/provision/preseed_default.erb#L19

When regenerating snapshots it's always a good idea to disable all plugins